testutil/singlevm/cleanup.sh

```diff
@@ -21,4 +21,7 @@ sudo docker rm -v -f ceph-demo
 sudo rm /etc/ceph/*
 sudo rm /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
 sudo rm /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
+sudo rm /var/lib/ciao/images/4e16e743-265a-4bf2-9fd1-57ada0b28904
```

Comment:
Cephfs is not enabled in Single VM.  There's no point as there is only one machine so we don't need a network file system and enabling it will just slow things down.  The image service doesn't care whether cephfs is running or not.

For the sake of experimentation I did try enabling cephfs in Single VM locally.  It does work but it slows down the setup.sh script.  In particular you need to wait 10 seconds after you start the  ceph docker container before you can mount cephfs.